packages/components/date-picker/src/date-picker-com/panel-month-range.vue

```diff
@@ -184,6 +184,7 @@ const handleRangePick = (val: RangePickValue, close = true) => {
   if (maxDate.value === maxDate_ && minDate.value === minDate_) {
     return
   }
+  emit('calendar-change', [minDate_.toDate(), maxDate_ && maxDate_.toDate()])
```

Comment:
> calendar-change | triggers when the calendar selected date is changed. Only for daterange

IMO, The description in the document may need to be updated.

Reply:
opps, the committer never come up, could you change the doc later?

Reply:
Thanks for guys' help, I had a vacation last week, didn't notice here.